packages/core/src/ceramic.ts

```diff
@@ -569,8 +570,8 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
     }
 
     if (EnvironmentUtils.useRustCeramic()) {
-      this._logger.warn(
-        `Running Ceramic using rust-ceramic. If you want to use Ipfs, run with IPFS_FLAVOR=go.`
+      this._logger.imp(
+        `Running Ceramic using ceramic-one (rust-ceramic). To use IPFS (kubo), specify a --ipfs-url or run with IPFS_FLAVOR=go.`
```

Comment:
is this message right?  I don't think just specifying `--ipfs-url` makes you use kubo, if the url you specify points at a ceramic-one node.

TBH I would vote we just remove all reference to ipfs/kubo/go entirely in the default path.  If they're already on ceramic-one then they're where we want them, why tell them how to move backwards?

I would, however, like a warning if you're still on kubo that you're on a deprecated config and should move forward to ceramic-one.